packages/oui-datagrid/src/datagrid.controller.js

```diff
@@ -150,6 +151,17 @@ export default class DatagridController {
         }
     }
 
+    toggleSubrows (index) {
+        this.expandedLines[index] = !this.expandedLines[index];
+    }
+
+    hasSubrows (row) {
+        if (!row) {
+            return false;
+        }
+        return this.subrows && row[this.subrows] && row[this.subrows].length;
```

Comment:
`!_.isEmpty(row[this.subrows])` ?

For a function called `hasSubrows`, it doesn't return a boolean but a number. It seems weird.